static/js/markdown_config.js

```diff
@@ -77,6 +77,7 @@ export const get_helpers = () => ({
 
     // settings
     should_translate_emoticons: () => user_settings.translate_emoticons,
+    should_display_emoji_reaction_users: () => user_settings.display_emoji_reaction_users,
```

Comment:
I don't think this line is correct -- we don't use this property in the Markdown processor.